### PR TITLE
Feature/welsh matei

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6585

### Change description ###
Welsh translations

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-basic-access-view/case-basic-access-view.component.html [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
-      <dd class="govuk-summary-list__value">{{courtOrHearingCentre | titlecase}}</dd>
+      <dt class="govuk-summary-list__key summary-key">{{'Court or hearing centre' | rpxTranslate}}</dt>
+      <dd class="govuk-summary-list__value">{{courtOrHearingCentre | rpxTranslate | titlecase}}</dd>
     </div>
 
     <div class="govuk-summary-list__row" *ngIf="accessType">
-      <dt class="govuk-summary-list__key summary-key">Access</dt>
-      <dd class="govuk-summary-list__value">{{accessType | titlecase}}</dd>
+      <dt class="govuk-summary-list__key summary-key">{{'Access' | rpxTranslate}}</dt>
+      <dd class="govuk-summary-list__value">{{accessType | rpxTranslate | titlecase}}</dd>
```
Comment (rayliangatos) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-viewer/case-basic-access-view/case-basic-access-view.component.html:32]: suppose we need to call the titlecase pipe first then rpxTranslate?
```suggestion
      <dd class="govuk-summary-list__value">{{accessType | titlecase | rpxTranslate }}</dd>
```

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.html [hunk 2]
```diff
@@ -1,27 +1,27 @@
 <form  (ngSubmit)="apply()">
   <div class="form-group">
-    <label class="form-label" for="cc-jurisdiction">Jurisdiction</label>
+    <label class="form-label" for="cc-jurisdiction">{{'Jurisdiction' | rpxTranslate}}</label>
     <select class="form-control ccd-dropdown" id="cc-jurisdiction" name="jurisdiction" [formControl]="filterJurisdictionControl" (change)="onJurisdictionIdChange()">
-      <option value="">--Select a value--</option>
-      <option *ngFor="let j of jurisdictions" [value]="j.id">{{j.name}}</option>
+      <option value="">{{'--Select a value - -' | rpxTranslate}}}}</option>
```
Comment (rayliangatos) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.html:5]: suppose we don't translate any signs but only the 'Select a value', and some other issues are '- -', and 4 '}}}}'?
```suggestion
      <option value="">--{{'Select a value' | rpxTranslate}}--</option>
```

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.html [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
     </select>
   </div>
 
   <div class="form-group">
-    <label class="form-label" for="cc-case-type">Case type</label>
+    <label class="form-label" for="cc-case-type">{{'Case type' | rpxTranslate}}</label>
     <select class="form-control ccd-dropdown" id="cc-case-type" name="case-type" [formControl]="filterCaseTypeControl" (change)="onCaseTypeIdChange()">
-      <option value="">--Select a value--</option>
-      <option *ngFor="let ct of selectedJurisdictionCaseTypes" [value]="ct.id">{{ct.name}}</option>
+      <option value="">{{'--Select a value--' | rpxTranslate}}</option>
```
Comment (rayliangatos) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.html:13]: same as above
```suggestion
      <option value="">--{{'Select a value' | rpxTranslate}}--</option>
```

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.html [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
     </select>
   </div>
 
   <div class="form-group">
-    <label class="form-label" for="cc-event">Event</label>
+    <label class="form-label" for="cc-event">{{'Event' | rpxTranslate}}}}</label>
     <select class="form-control ccd-dropdown" id="cc-event" name="event" [formControl]="filterEventControl" (change)="onEventIdChange()">
-      <option value="">--Select a value--</option>
-      <option *ngFor="let e of selectedCaseTypeEvents" [value]="e.id">{{e.name}}</option>
+      <option value="">{{'--Select a value--' | rpxTranslate}}</option>
```
Comment (rayliangatos) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/create-case-filters/create-case-filters.component.html:21]: same as above

**Review by maliumer — APPROVED** [review]

**Pull request merged**